server/errors/errors.go

```diff
@@ -83,8 +84,14 @@ var (
 	ErrNoKeyAlgorithm = errcode.Register(errGroup, errcode.ErrorDescriptor{
 		Value:          "NO_KEYALGORITHM",
 		Message:        "The server does not have a key algorithm configured.",
-		Description:    "No key algorihtm has been configured for the server and it has been asked to perform an operation that requires generation.",
+		Description:    "No key algorithm has been configured for the server and it has been asked to perform an operation that requires generation.",
 		HTTPStatusCode: http.StatusInternalServerError,
 	})
+	ErrCannotRotateKey = errcode.Register(errGroup, errcode.ErrorDescriptor{
```

Comment:
Given there are lots of reasons a key rotation can fail, should the name be more specific like `ErrKeyRotationRateLimited`?